packages/data-stores/src/help-center/actions.ts

```diff
@@ -128,17 +128,24 @@ export const setUserDeclaredSite = ( site: SiteDetails | undefined ) =>
 		site,
 	} as const );
 
+export const resetStore = () =>
+	( {
+		type: 'HELP_CENTER_RESET_STORE',
+	} as const );
+
 export const startHelpCenterChat = function* ( site: HelpCenterSite, message: string ) {
-	yield setInitialRoute( '/inline-chat' );
+	yield setInitialRoute( '/contact-form?mode=CHAT' );
```

Comment:
I know this is not the same as what `inline-chat` offered, but `inline-chat` has been removed, and this action is only used in the dead `client/me/help/help-contact` form (that I will be removing very soon).